services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java

```diff
@@ -365,9 +381,11 @@ long installOneSwitchFlow(final DatapathId dpid, final String flowId, final Long
      *
      * @param dpid switch id.
      * @param multiTable flag
+     * @param switchLldp flag
```

Comment:
Please, add more information about the flag. "switchLldp" is controversial and unclear by the purpose.

Reply:
fixed